src/openfermion/functionals/get_one_norm.py

```diff
@@ -0,0 +1,117 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""
+Function to calculate the 1-Norm of a molecular Hamiltonian
+(in the restricted formalism) after fermion-to-qubit transformation
+from a MolecularData class. See https://arxiv.org/abs/2103.14753
+for more information on the 1-norm.
+"""
+
+import numpy as np
+
+
+def get_one_norm_mol(molecule):
+    r"""
+
+    Returns the 1-Norm of a RHF or ROHF Hamiltonian described in
+    https://arxiv.org/abs/2103.14753 after a fermion-to-qubit
+    transformation given nuclear constant, one-body (2D np.array)
+    and two-body (4D np.array) integrals in spatial orbital basis.
+
+    Parameters
+    ----------
+
+    molecule : MolecularData class representing a molecular Hamiltonian
+
+    Returns
+    -------
+    one_norm : 1-Norm of the qubit Hamiltonian
+    """
+    return get_one_norm_int(molecule.nuclear_repulsion,
+                            molecule.one_body_integrals,
+                            molecule.two_body_integrals)
+
+
+def get_one_norm_mol_woconst(molecule):
```

Comment:
Okay this is the very last thing and then this is done.  It seems that now that we aren't dispatching these functions need to be properly documented.  You  You can just reference `get_one_norm_int` or specify that the parameters molecule: molecularData class.

Reply:
I think its fixed now. Thank you for being precise Nick, it was a learning experience for me :)